src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1642,6 +1643,14 @@ public static LivingEntity getLivingEntity(ModElement modElement, Random random,
 		livingEntity.modelHeight = 1.3;
 		livingEntity.mountedYOffset = -3.1;
 		livingEntity.modelShadowSize = 1.8;
+		if (!emptyLists) {
+			livingEntity.entityDataEntries.add(
+					new PropertyDataWithValue<>(new PropertyData.LogicType("Logic"), _true));
+			livingEntity.entityDataEntries.add(
+					new PropertyDataWithValue<>(new PropertyData.IntegerType("Number"), random.nextInt()));
+			livingEntity.entityDataEntries.add(new PropertyDataWithValue<>(new PropertyData.StringType("String"),
+					getRandomItem(random, new String[] { "Number", "Logic", "String" })));

```

Comment:
This could confuse someone checking the code about the intent of those values in String[], probably better have "value1", "value2", "value3" here